### PR TITLE
docs: update usage in README and add test for it

### PR DESCRIPTION
**Pull request by hi-rustin**
1. Updated the usage in the readme.
2. Added trycmd to verify it in tests.

[rendered](https://github.com/hi-rustin/cargo-information/blob/rustin-patch-readme-usage/README.md)

### CODE REVIEW

**Review by hi-rustin — COMMENTED** [review]
🔢 Self-check

The only thing I am a little worried about is that we introduced another test framework to the project.

File: tests/testsuite/cargo_information/readme.rs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#[test]
+// You can use `TRYCMD=overwrite` to overwrite the expected output.
+fn readme() {
```
Comment (epage) [inline tests/testsuite/cargo_information/readme.rs:3]: This should be in its own test binary

Comment (hi-rustin) [reply tests/testsuite/cargo_information/readme.rs]: Make sense.

Comment (hi-rustin) [reply tests/testsuite/cargo_information/readme.rs]: Thanks for your review!  💚 💙 💜 💛 ❤️

**Review by hi-rustin — COMMENTED** [review]
🔢 Self-check

**Pull request merged**